### PR TITLE
Simple parsing nginx metrics [parseStubStats]

### PR DESCRIPTION
**Pull request by mirecl**
### Proposed changes

Simple visual templating Nginx for parsing metrics.

### Checklist

- [X] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-prometheus-exporter/blob/master/CONTRIBUTING.md) guide
- [X] I have proven my fix is effective or that my feature works
- [X] I have checked that all unit tests pass after adding my changes
- [X] I have ensured the README is up to date
- [X] I have rebased my branch onto master
- [X] I will ensure my PR is targeting the master branch and pulling from my branch on my own fork

### CODE REVIEW

File: client/nginx.go [hunk 1]
```diff
@@ -66,79 +71,26 @@ func (client *NginxClient) GetStubStats() (*StubStats, error) {
...
+	var s StubStats
+	if _, err := fmt.Fscanf(r, templateMetrics,
+		&s.Connections.Active,
+		&s.Connections.Accepted,
+		&s.Connections.Handled,
+		&s.Requests,
+		&s.Connections.Reading,
+		&s.Connections.Writing,
+		&s.Connections.Waiting); err != nil {
+		return nil, err
```
Comment (soneillf5) [inline client/nginx.go:93]: It's 'good' Golang to wrap en error from the location it was found, this helps us trace an error in the logs. 
Can you change `return nil, err`
to something like `return fmt.Errorf("messge", err)`
similar to the other errors in this file?

**Review by soneillf5 — APPROVED** [review]
Looks great thanks. I've added a minor comment but it's optional.

File: client/nginx.go (hunk 1 above)
Comment (mirecl) [reply client/nginx.go]: No problem @soneillf5 , done!

**Comment by lucacome** [comment]
@mirecl could you please rebase your branch against master?

**Comment by mirecl** [comment]
@lucacome , done.

> @mirecl could you please rebase your branch against master?

**Review by lucacome — APPROVED** [review]

**Pull request merged**